backend/app/Services/FileParser.php

```diff
@@ -98,90 +112,47 @@ private function searchTmdb($title, $item)
     public function foundInDatabase($value, $type)
     {
       if($type == 'title') {
-        return $this->item->searchTitle($value)->first();
+        return $this->item->findByTitle($value)->first();
       }
 
-      return $this->item->searchTmdbId($value)->first();
+      return $this->item->findByTmdbId($value)->first();
     }
 
     /**
      * Check which status the file has.
      * Create new src if the status is 'added'.
-     * Update src if status is 'update'.
-     * Remove src if status is 'remove'.
+     * Update src if status is 'updated'.
+     * Remove src if status is 'removed'.
      *
      * @param $item
      * @param $tmdb_id
      * @return \Exception|mixed
      */
-    public function handleStatus($item, $tmdb_id)
+    public function handleStatus($item, $tmdbId)
     {
       if($item->status == 'added') {
-        return $this->storeSrc($item, $tmdb_id);
-      }
-
-      if($item->status == 'update') {
-        //
-      }
-
-      if($item->status == 'removed') {
-        //
+        return $this->storeSrc($item, $tmdbId);
       }
 
       return new \Exception('No status in file found');
```

Comment:
You sure you wanna throw an exception? What  if file parser gets an update and forget to update flox? Some changes are getting saved into db and then exception? How about a rollback of transactions?